src/App.js

```diff
@@ -1,24 +1,16 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import DisplayBooks from './component/DisplayBooks';
+import Category from './component/Category';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<DisplayBooks />} />
+        <Route path="/Category" element={<Category />} />
```

Comment:
Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 🤓